astropy/units/format/unicode_format.py

```diff
@@ -43,7 +52,7 @@ def _format_unit_power(cls, unit, power=1):
         return name
 
     @classmethod
-    def _format_superscript(cls, number):
+    def _format_superscript(cls, number: str) -> str:
         mapping = str.maketrans(
```

Comment:
Unrelated (though perhaps this PR is as good a place as any), but this dict should be moved to outside this function.

Reply:
Pull requests that add type annotations should not include any code changes.